assets/stylesheets/shared/_color-schemes.scss

```diff
@@ -58,6 +58,8 @@
 	--color-accent-900: #{$muriel-hot-pink-900};
 	--color-accent-900-rgb: #{hex-to-rgb( $muriel-hot-pink-900 )};
 
+	--color-white: #{ $muriel-white };
```

Comment:
Is there any way we could prevent using a color name in the CSS variable here? My concern would be us introducing a precedent for others introducing similar variables. If --color-white needs to be black in a different scheme that would be not ideal.

Reply:
we already have neutral-0, so i'm not sure where to go with it... neutral-00? The bigger issue is that usage is going to vary wildly. Some of the cases covered here are white-fill-on-color-fill-background, while others are white-text-on-neutral-dark-background. I don't think those would totally flip when moving to dark... We'll probably have to readdress this anyway on a case by case basis when doing the dark theme.